kiwi/builder/disk.py

```diff
@@ -642,17 +526,243 @@ def create_install_media(self, result_instance: Result) -> Result:
 
         return result_instance
 
+    def _raid_instance(self, device_map: Dict) -> RaidDevice:
+        return RaidDevice(device_map['root'])
+
+    def _luks_instance(self, device_map: Dict) -> LuksDevice:
+        return LuksDevice(device_map['root'])
```

Comment:
The reason why I put the construction of instances into an extra method is that I expect those to receive more parameters in the future. When we change to context managers this will result in a nested structure of "with" statements and in contrast to today's implementation there will then be more object constructions of the same class. If it still stays that simple after the refactoring into context managers I will delete the one liner instance creation methods again. It will not hurt because they "sort of" private with the "_" and will not be exposed to the public API, at least not from a documentation perspective